src/lowering/desugar.ml

```diff
@@ -214,7 +214,8 @@ and exp' at note = function
     I.PrimE (I.OtherPrim "blob_size", [exp e1])
   (* Normal call *)
   | S.CallE (par_opt, e1, inst, e2) ->
```

Comment:
Isn't it weird to just drop it? It might have a side-effect?

Reply:
76b40cfb4dbac5bb6b2bf291915bad5c92ddfed0 fixes the effects issue.